ospf6d/ospf6d.c

```diff
@@ -847,9 +847,15 @@ DEFUN(show_ipv6_ospf6_database_adv_router_linkstate_id,
 
 	for (ALL_LIST_ELEMENTS_RO(om6->ospf6, node, ospf6)) {
 		if (all_vrf || strcmp(ospf6->name, vrf_name) == 0) {
-			ospf6_lsdb_type_show_wrapper(vty, level, NULL, &id,
+			type = OSPF6_SCOPE_AREA;
+			ospf6_lsdb_type_show_wrapper(vty, level, &type, &id,
```

Comment:
Need to loop through all the LSA types and not based on the scope here.

Reply:
We just need to change `ospf6_lsdb_type_show_wrapper` to `ospf6_lsdb_show_wrapper`.